tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotSerializer.NotCapturedReason.cs

```diff
@@ -16,7 +18,7 @@ private enum NotCapturedReason
             fieldCount,
             timeout,
             redactedIdent,
-            redactedType
+            redactedType,
```

Comment:
nitpick: changed this file by mistake?

Reply:
The initial implementation involved a change in the file, a comma was accidentally left. I'll fix it.